plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -115,6 +122,14 @@ const renderTabs = (
 					key={ tabKey }
 				>
 					{ tabs[ tabKey ]?.title }
+					{ tabs[ tabKey ]?.showUpdateCount && updateCount > 0 && (
+						<span
+							className="woocommerce-marketplace__update-count"
+							dangerouslySetInnerHTML={ sanitizeHTML(
+								'<span>' + updateCount + '</span>'
+							) }
```

Comment:
Wouldn't this work?
```suggestion
<span> { updateCount } </span>
```

Reply:
@denho JSX won't work here as a string needs to be passed into `sanitizeHTML`

Reply:
@thilinah This worked for me. Why does it have to be passed to `sanitizeHTML`?

Reply:
Updated here (https://github.com/woocommerce/woocommerce/pull/46088/commits/28ebf869379c05063ded58e57abb2ccef0f0d9a2), didn't notice we actually don't need to dangerouslySetInnerHTML.